app/views/layouts/_nav_menu.html.erb

```diff
@@ -1,25 +1,29 @@
 <div class="crayons-dropdown left-2 right-2 s:left-auto p-0 crayons-header__menu__dropdown inline-block">
   <% if user_signed_in? %>
-    <div id="user-profile-link-placeholder" class="border-0 border-b-1 border-solid border-base-20 p-1">
-      <a id="first-nav-link" class="crayons-link crayons-link--block">
-        <div>
-          <span class="fw-medium block">...</span>
-          <small class="fs-s color-base-50"></small>
-        </div>
-      </a>
-    </div>
+    <ul class="p-0" id="crayons-header__menu__dropdown__list">
+      <li id="user-profile-link-placeholder" class="border-0 border-b-1 border-solid border-base-20 p-1">
+        <a id="first-nav-link" class="crayons-link crayons-link--block">
+          <div>
+            <span class="fw-medium block">...</span>
+            <small class="fs-s color-base-50"></small>
+          </div>
+        </a>
+      </li>
 
-    <div class="border-0 border-b-1 border-solid border-base-20 p-1">
-      <a href="<%= admin_path %>" class="crayons-link crayons-link--block js-header-menu-admin-link hidden" data-no-instant>Admin</a>
-      <a href="<%= dashboard_path %>" class="crayons-link crayons-link--block">Dashboard</a>
-      <a href="<%= new_path %>" class="crayons-link crayons-link--block">Write a Post</a>
-      <a href="<%= readinglist_path %>" class="crayons-link crayons-link--block">Reading list</a>
-      <a href="<%= user_settings_path %>" class="crayons-link crayons-link--block">Settings</a>
-    </div>
-    <div class="p-1">
-      <a href="<%= signout_confirm_path %>" class="crayons-link crayons-link--block" id="last-nav-link">Sign Out</a>
-    </div>
+      <li class="px-1 js-header-menu-admin-link data-no-instant hidden">
```

Comment:
`data-no-instant` was an attribute before. Is this turned into a class intentionally?

Reply:
Ah, no! Good catch. And I'm guessing that should data attribute should probably stay on the link for InstantClick, so I'll replace it. Thanks!